File: Python Word Examples/TablesApiDemo/Table/updateTableFormat.py
import spirecloudword
from spirecloudword import TableFormat, Border, Color, Borders
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.TablesApi(configuration)

file_name = 'Table.docx'
index = 0
node_path = 'Section/0/Body/0'
properties = TableFormat()
properties._bidi = True
border = Border()
border._border_type = 'ThinThickThinMediumGap'
border._color = Color(255, 25, 1)
border._line_width = 10
properties._borders = Borders(border)
extra = {'folder': "input", 'dest_file_path': 'output/UpdateTableFormat_3.docx'}
result =api.update_table_format(file_name, node_path, index, properties, **extra)

File: JavaScript Word Examples/TablesApiDemo/Table/updateTableFormat.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.TablesApi(apiClient); 

    var filename = "Table.docx";
    var index = 0;
    var nodePath = "Section/0/Body/0";
    var format = new Spirecloudword.TableFormat();
    format.bidi=true;
    format.borders = new Spirecloudword.Borders();
    format.borders.topBorder=new Spirecloudword.Border();
    format.borders.topBorder.borderType="Dot"; //Available types: Dot, Double, ThinThickThinMediumGap
    format.borders.topBorder.color=new Spirecloudword.Color();
    format.borders.topBorder.color.red=255;
    format.borders.topBorder.color.green=25;
    format.borders.topBorder.color.blue=1;
    format.borders.topBorder.lineWidth=3;
    var opt = {folder: 'Tables'};
    var destFilePath = "output/Tables/UpdateTableFormat_1.docx";
    instance.updateTableFormat(filename, nodePath, index, format, destFilePath, opt,
    function (error, data, response) {
         if (error) throw error;
    });
})();